.github/workflows/audit.yml

```diff
@@ -0,0 +1,35 @@
+name: Audit
+
+on:
+  push:
+    branches:
+      - '**'
+    tags-ignore:
+      - '**'
+    paths:
+      - 'Cargo.lock'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    env:
+      DB_URL: http://neo4j:testpass@localhost:7474/db/data
+      DB_PASS: testpass
+      
+    steps:
+    - uses: actions/checkout@v2
+      
+    - name: Install Cargo Audit
+      run: ls -la rustbins/bin/cargo-audit | grep cargo-audit || cargo install cargo-audit --root ./rustbins
+    
+    - name: Run Cargo Audit
+      run: rustbins/bin/cargo-audit audit || echo "Audit failed!"
+        
+    - name: Create security issue if audit fails
+      if: failure()
+      uses: JasonEtco/create-an-issue@v2
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        title: Cargo audit failure
```

Comment:
It turns out that this github job as written will not have the intended effect. Short answer is it will not deduplicate. I tried to account for this by only running this job when a library changes (by looking at Cargo.lock), but it is now obvious to me that that approach will no work in situations where a new vuln is reported. I'm going to remove this job from this PR and add it to another ticket.